ui/modal/info_modal.go

```diff
@@ -125,17 +129,32 @@ func (in *InfoModal) SetupWithTemplate(template string) *InfoModal {
 	return in
 }
 
+func (in *InfoModal) handleEnterKeypress() {
+	select {
+	case event := <-in.keyEvent:
+		if (event.Name == key.NameReturn || event.Name == key.NameEnter) && event.State == key.Press {
+			in.enterKeyPressed = true
+			return
+		}
+	default:
+	}
+}
+
 func (in *InfoModal) Handle() {
+	in.handleEnterKeypress()
 
-	for in.btnPositve.Button.Clicked() {
+	if in.btnPositve.Button.Clicked() || in.enterKeyPressed {
 		in.DismissModal(in)
+		in.enterKeyPressed = false
 		in.positiveButtonClicked()
 	}
 
 	for in.btnNegative.Button.Clicked() {
 		in.DismissModal(in)
 		in.negativeButtonClicked()
 	}
+
+	in.RefreshWindow()
```

Comment:
What does this do?

Reply:
It refreshes the window after the enter button has been clicked. If the window is not refreshed, whenever the enter key is presed it does not perform the action until the mouse cursor is moved